api/docker/Dockerfile

```diff
@@ -25,4 +25,4 @@ VOLUME ["/hygieia/logs"]
 
 EXPOSE 8080
 CMD ./properties-builder.sh &&\
-  java -Djava.security.egd=file:/dev/./urandom -jar api.jar --spring.config.location=/hygieia/dashboard.properties
+  java -Djava.security.egd=file:/dev/./urandom -Djasypt.encryptor.password=$JASYPT_ENCRYPTOR_SECRET -jar api.jar --spring.config.location=/hygieia/dashboard.properties
```

Comment:
@jmcshane yes the **JASYPT_ENCRYPTOR_SECRET** would be an environment property and this is reflected in the updates I made on api/README.md. Please go over the diff on the readme and let me know if there's anything lacking.